### PR TITLE
Add minor changes to CompareGraphs.py

### PR DESCRIPTION
**Pull request by stefanopolitano**
Hi! I changed the config. files, but I forgot to commit the changes to CompareGraphs

### CODE REVIEW

File: comparisons/CompareGraphs.py [hunk 1]
```diff
@@ -83,7 +86,13 @@
...
     else:
         hToCompare[iFile].SetName(f'g{iFile}')
-    SetObjectStyle(hToCompare[iFile], color=GetROOTColor(color), markerstyle=GetROOTMarker(marker), fillstyle=0)
+    SetObjectStyle(hToCompare[iFile],
+                   color=GetROOTColor(color),
+                   markerstyle=GetROOTMarker(marker),
+                   markersize=inputCfg['options']['markersize'],
+                   linewidth=inputCfg['options']['linewidth'],
+                   fillstyle=inputCfg['options']['fillstyle'][iFile], 
+                   fillalpha=inputCfg['options']['fillalpha'][iFile])
```
Comment (fcatalan92) [inline comparisons/CompareGraphs.py:95]: I would access the dictionary only once defining the variables as done for the others

**Review by fcatalan92 — CHANGES_REQUESTED** [review]

File: comparisons/CompareGraphs.py (hunk 1 above)
Comment (stefanopolitano) [reply comparisons/CompareGraphs.py]: Hi @fcatalan92! Now it should be ok. Let me know if there is still something wrong!

File: comparisons/CompareGraphs.py [hunk 2]
```diff
@@ -83,7 +92,13 @@
...
     else:
         hToCompare[iFile].SetName(f'g{iFile}')
-    SetObjectStyle(hToCompare[iFile], color=GetROOTColor(color), markerstyle=GetROOTMarker(marker), fillstyle=0)
+    SetObjectStyle(hToCompare[iFile],
+                   color=GetROOTColor(color),
+                   markerstyle=GetROOTMarker(marker),
+                   markersize=markersize,
+                   linewidth=linewidth,
+                   fillstyle=fillstyle[iFile],
+                   fillalpha=fillalpha[iFile])
```
Comment (fcatalan92) [inline comparisons/CompareGraphs.py:101]: Remove `[iFile]`

**Review by fcatalan92 — CHANGES_REQUESTED** [review]

File: comparisons/CompareGraphs.py (hunk 2 above)
Comment (stefanopolitano) [reply comparisons/CompareGraphs.py:101]: Hi @fcatalan92! Thank you for your comment. Actually, the idea behind the 'iFile' was to use different styles and transparencies for different Files. Indeed, when I wrote the code, I had to represent overlapping points with different error bars, hence it was useful to have this feature. However, I can change it.

Comment (fcatalan92) [reply comparisons/CompareGraphs.py:101]: But now you are already iterating over the lists in the for loop, so `fillstyle` and `fillalpha` are not list anymore, right?

Comment (stefanopolitano) [reply comparisons/CompareGraphs.py:101]: Yes, you're right. Sorry, I fix it.

**Review by fcatalan92 — APPROVED** [review]

**Pull request merged**